frontend/src/metabase-lib/lib/queries/StructuredQuery.js

```diff
@@ -1350,6 +1362,15 @@ export default class StructuredQuery extends AtomicQuery {
     return db && db.hasFeature("nested-queries");
   }
 
+  cleanEmptyNesting(): StructuredQuery {
```

Comment:
Looks like `cleanNesting` replaced this?

Reply:
Yup good catch